app/pb_kits/playbook/pb_user_badge/_user_badge.jsx

```diff
@@ -4,18 +4,18 @@ import React from 'react'
 type UserBadgeProps = {
   className?: String,
   id?: String,
-  badge?: 'million-dollar' | 'veteran' ,
+  badge: React.ReactNode,
   size?: 'sm' | 'md' | 'lg',
 }
 
 const UserBadge = ({
-  badge = 'million-dollar',
+  badge,
   size = 'md',
-}) => (
+}: UserBadgeProps) => (
   <div className={`pb_user_badge_kit_${size}`}>
-    <If condition={UserBadgeProps.badge == "veteran"}>
```

Comment:
You error was coming from `UserBadgeProps` attempting to be directly accessed. It is simply `badge` instead.

Reply:
Oh okay, I wasn't aware that badge would be accessible like that. Good to know